### ISSUE TITLE
Atlas postpacking not working with filtering or denoising with multiple-material objects

### ISSUE THREAD

**Issue by Naxela** [Naxela/The_Lightmapper#104]

**Comment by 23Mounir23** [comment]
yes this is right i wanted to add this issue today with multiple-material denoising not working and the lightmaps doesnot be applied on the materials

**Comment by Naxela** [comment]
It should be solved in https://github.com/Naxela/The_Lightmapper/commit/1a17d1cf3fc1e1940de3ab5b696f72f32b777daf
